src/query/scan_manager.c

```diff
@@ -7561,6 +7562,7 @@ resolve_domains_on_list_scan (LLIST_SCAN_ID * llsidp, val_list_node * ref_val_li
   for (scan_regu = llsidp->rest_regu_list; scan_regu != NULL; scan_regu = scan_regu->next)
     {
       if ((TP_DOMAIN_TYPE (scan_regu->value.domain) == DB_TYPE_VARIABLE
+	   || (TP_DOMAIN_TYPE (scan_regu->value.domain) == DB_TYPE_NUMERIC && scan_regu->value.domain->precision == 0)
```

Comment:
|| (TP_DOMAIN_TYPE (scan_regu->value.domain) == DB_TYPE_NUMERIC && scan_regu->value.domain->precision == 0)
```suggestion
	   || (TP_DOMAIN_TYPE (scan_regu->value.domain) == DB_TYPE_NUMERIC && scan_regu->value.domain->precision == DB_NUMERIC_PRECISION_ANY)
```